x/evm/keeper/grpc_query.go

```diff
@@ -380,14 +380,10 @@ func (k Keeper) EthCall(c context.Context, req *types.EthCallRequest) (*types.Ms
 	ethCfg := params.ChainConfig.EthereumConfig(k.eip155ChainID)
 
 	evm := k.NewEVM(msg, ethCfg, params)
-	res, err := k.ApplyMessage(evm, msg, ethCfg)
+	res, err := k.ApplyMessage(evm, msg, ethCfg, true)
```

Comment:
add comment on why we are using simulate

Reply:
done